src/OrbitGl/App.cpp

```diff
@@ -1399,10 +1399,17 @@ void OrbitApp::LoadSymbols(const std::filesystem::path& symbols_path, ModuleData
 
 orbit_base::Future<ErrorMessageOr<void>> OrbitApp::LoadPresetModule(
     const std::string& module_path, const orbit_client_protos::PresetModule& preset_module) {
+  if (!GetTargetProcess()->IsModuleLoaded(module_path)) {
+    return {ErrorMessage{"Module not loaded by process."}};
```

Comment:
BTW, The additional curly braces are not needed anymore. Future now has a special constructor which takes an error message.